src/renderer/components/ft-share-button/ft-share-button.js

```diff
@@ -59,6 +60,7 @@ export default Vue.extend({
     },
 
     copyInvidious() {
+      FtToastEvents.$emit('toast.open', 'Invidious URL copied to clipboard')
```

Comment:
I would like to change the syntax in how this works slightly.  We can keep what you have so far and these changes will be building on top of them.

Let's move the functionality of opening toast messages over to `src/renderer/store/utils/js`.  This will give the other stores access to making messages instead of just components.  Having a simple `showToast` function that calls `      FtToastEvents.$emit('toast.open', 'message')` is good.  This function can include any parameters needed (like `callback` and `duration`) to send over.  Within a component, the syntax would change to sending over the function as a `mapAction` and the syntax would then change to `this.showToast()`.  You can take a look at the `general-settings` component for a reference on how this works.

Reply:
I've now added showToast() to the actions in src/renderer/store/module/utils.js. I then import this using mapAction in the share button.